server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -374,58 +404,26 @@ private CommandDispatcher getDispatcher(CommandClass commandClass) {
         return dispatcherRegistry.getDispatcher(commandClass);
     }
 
+    private static CommandScheduler createCommandScheduler() {
+        @SuppressWarnings("AccessOfSystemProperties")
+        final String appEngineVersion = System.getProperty("com.google.appengine.runtime.version");
```

Comment:
This needs to be a feature of a utility class, which is visible to the whole framework. It can be called `Environment` or something like that.